server/lib/middleware.js

```diff
@@ -21,6 +21,7 @@ const ASSERT_TOKEN = env.get("ASSERT_TOKEN");
 
 module.exports = function middlewareConstructor(config) {
   let cryptr = new Cryptr(env.get("SESSION_SECRET"));
+  let cryptrFallback = new Cryptr(env.get("SESSION_SECRET"), "aes-256");
```

Comment:
Let's add a comment here with a link to https://github.com/MauriceButler/cryptr/compare/fabae97a61119d69f03fc189f7c95dda826c96b7...master#diff-168726dbe96b3ce427e7fedce31bb0bcR9 so it's clear why we're doing this.